core/commons/che-core-commons-tracing/src/main/java/org/eclipse/che/commons/tracing/TracingTags.java

```diff
@@ -26,6 +26,19 @@
   public static final AnnotationAwareStringTag MACHINE_NAME =
       new AnnotationAwareStringTag("machine.name");
 
+  /** The id of the user the span is related to. */
+  public static final AnnotationAwareStringTag USER_ID = new AnnotationAwareStringTag("user.id");
+
+  /** The id of the stack the span is related to. */
+  public static final AnnotationAwareStringTag STACK_ID = new AnnotationAwareStringTag("stack.id");
+
+  /**
+   * The entity that stopped workspace, which can be either user ID, or name of component that
+   * stopped it (e.g. activity checker) .
+   */
+  public static final AnnotationAwareStringTag STOPPED_BY =
+      new AnnotationAwareStringTag("stopped.by");
```

Comment:
I'm not aware of annotation names convention. But here is my felling:
`user.id` and `stack.id` looks logic for me. But I prefer `stopped_by` instead of `stopped.by`. Could we do it? @metlos WDYT?

Reply:
> I prefer `stopped_by` instead of `stopped.by`. Could we do it? @metlos WDYT?

:+1:

Reply:
ok, changing to it then